argocd/resource_argocd_application_test.go

```diff
@@ -261,6 +266,32 @@ func TestAccArgoCDApplication_IgnoreDifferences(t *testing.T) {
 				ImportStateVerify:       true,
 				ImportStateVerifyIgnore: []string{"wait", "cascade", "status"},
 			},
+			{
+				Config: testAccArgoCDApplicationIgnoreDiffManagedFieldsManagers(
+					acctest.RandomWithPrefix("test-acc")),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet(
+						"argocd_application.ignore_differences_managed_fields_managers",
+						"metadata.0.uid",
+					),
+					resource.TestCheckResourceAttr(
+						"argocd_application.ignore_differences_managed_fields_managers",
+						"spec.0.ignore_difference.0.managed_fields_managers.0",
+						"some-controller-owner",
+					),
+					resource.TestCheckResourceAttr(
+						"argocd_application.ignore_differences_managed_fields_managers",
+						"spec.0.ignore_difference.1.managed_fields_managers.1",
+						"some-other-controller-owner",
+					),
+				),
+			},
+			{
+				ResourceName:            "argocd_application.ignore_differences_managed_fields_managers",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"wait", "cascade", "status"},
```

Comment:
``` suggestion
				ImportStateVerifyIgnore: []string{"wait", "cascade", "status", "validate"},
```

Need to add `validate` in here to fix tests after #411 was merged

Reply:
I was wondering if this was what I needed to fix the tests! Thank you. I was struggling to figure out where `validate` was coming from. When I was building the provider on local I saw no trace of this, so thanks for clearing this up!